src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -110,6 +110,12 @@ def _get_feature_in_preview_message():
 
 register_cli_argument('acs', 'disable_browser', help='Do not open browser after opening a proxy to the cluster web user interface')
 register_cli_argument('acs dcos browse', 'name', name_arg_type)
+register_cli_argument('acs dcos browse', 'ssh_key_file',
+                      required=False,
+                      help='Path to an SSH key file to use.',
+                      type=file_type,
+                      default=os.path.join('~', '.ssh', 'id_rsa'),
```

Comment:
Is this the appropriate default on Linux and Windows?  I'd be surprised if they weren't different.

Reply:
we already adopted this style for other ssh keys. In the same file, you can check how we default ssh-key-value. 

Btw, in the issue, it is the desired default per the discussion.